### PR TITLE
fzf 0.9.7

### PR DESCRIPTION
**Pull request by junegunn**
Thank you!

### CODE REVIEW

File: Library/Formula/fzf.rb [hunk 1]
```diff
@@ -3,9 +3,10 @@
 
 class Fzf < Formula
   homepage "https://github.com/junegunn/fzf"
-  version "0.9.4-1"
-  url "https://github.com/junegunn/fzf/archive/0.9.4-1.tar.gz"
-  sha1 "cab72a79d9b7776d0a441d0ae1ba679893df8a55"
+  version "0.9.7"
```
Comment (mikemcquaid) [inline Library/Formula/fzf.rb]: Can remove this `version`

Comment (junegunn) [inline Library/Formula/fzf.rb]: :ok_hand:

**Comment by junegunn** [comment]
@mikemcquaid Removed `version`. Is there anything else you'd like to address?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**